app/mainAppWindow/index.js

```diff
@@ -88,7 +102,6 @@ function onBeforeRequestHandler(details, callback) {
 function onNewWindow(event, url, frame, disposition, options) {
 	if (url.startsWith('https://teams.microsoft.com/l/meetup-join')) {
 		event.preventDefault();
-		window.loadURL(url);
```

Comment:
Does sharing a meeting in within teams work? We used to need this for that use case. (maybe we don't need it  anymore)

Reply:
Yes, it does. I tested and worked